library/std/src/os/unix/process.rs

```diff
@@ -179,6 +179,9 @@ pub trait CommandExt: Sealed {
     /// ```
     #[stable(feature = "process_set_process_group", since = "1.64.0")]
     fn process_group(&mut self, pgroup: i32) -> &mut process::Command;
+
+    #[unstable(feature = "process_setsid", issue = "105376")]
+    fn setsid(&mut self) -> &mut process::Command;
```

Comment:
I think the `fn setsid(&mut self, setsid: bool) -> &mut Command` signature might be preferable - this is what we have for `create_pidfd` `force_quotes` and `async_pipes`.